src/shared/api/cbioportal-types.d.ts

```diff
@@ -0,0 +1,88 @@
+declare namespace cbio
```

Comment:
I wrote it to help myself understand the old api - we probably won't use it but I agree it's a good example.